### ISSUE TITLE
Use ACM to sync Tekton Resources to kf-ci-v1

### ISSUE THREAD

**Issue by jlewi** [kubeflow/testing#677]
Opening up this issue to track using ACM to sync tekton resources to the kf-ci-v1 cluster.

Here's a dump of all tekton resources in namespace autodeploy

We only have the following tekton resources
```
NAME                                       AGE
pipeline.tekton.dev/deploy-gcp-blueprint   31d
NAME                                   AGE
task.tekton.dev/cleanup-kubeflow-ci    32d
task.tekton.dev/deploy-gcp-blueprint   34d
task.tekton.dev/notebook-test          32d
```

[auto-deploy.tekton.resources.txt](https://github.com/kubeflow/testing/files/4749050/auto-deploy.tekton.resources.txt)

Here is a list of all resources

[auto-deploy.all.txt](https://github.com/kubeflow/testing/files/4749057/auto-deploy.all.txt)

Note we have a cronjob to cleanup kubeflow-ci-deployment.

These dumps are so we can see if end up losing any resources after setting up ACM; I can't recall if ACM prunes existing resources that it didn't create; I don't think so.

**Comment by jlewi** [comment]
Here are the resources for namespace `tektoncd`

[tektoncd.all.txt](https://github.com/kubeflow/testing/files/4749060/tektoncd.all.txt)
[tektoncd.tekton.txt](https://github.com/kubeflow/testing/files/4749063/tektoncd.tekton.txt)

**Comment by jlewi** [comment]
Fixed by #679
